src/Raven.Server/Documents/Sharding/Operations/MultiOperation.cs

```diff
@@ -3,33 +3,39 @@
 using System.Diagnostics;
 using System.Threading;
 using System.Threading.Tasks;
-using Raven.Client.Documents.Commands;
 using Raven.Client.Documents.Operations;
 using Raven.Client.Extensions;
+using Raven.Client.Http;
 
 namespace Raven.Server.Documents.Sharding.Operations;
 
-public class MultiOperation
+public abstract class MultiOperation
 {
     private readonly object _locker = new();
 
-    private readonly long _id;
-    private readonly ShardedDatabaseContext _context;
+    protected readonly long Id;
+    protected readonly ShardedDatabaseContext Context;
```

Comment:
let's rename this to `DatabaseContext` or `ShardedDatabaseContext` ... too many contexts :)

Reply:
done